lib/node_modules/@stdlib/math/base/special/acotf/test/test.js

```diff
@@ -0,0 +1,271 @@
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	x = largerPositive.x;
+	expected = largerPositive.expected;
+	for ( i = 0; i < x.length; i++ ) {
+		y = acotf( x[ i ] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[ i ]+'. E: '+e+'.' );
+		} else {
+			delta = abs( y - e );
+			tol = 1.2 * EPS * abs( e );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[ i ]+'. y: '+y+'. E: '+e+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the inverse cotangent for larger negative values', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	x = largerNegative.x;
+	expected = largerNegative.expected;
+	for ( i = 0; i < x.length; i++ ) {
+		y = acotf( x[ i ] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[ i ]+'. E: '+e+'.' );
+		} else {
+			delta = abs( y - e );
+			tol = 1.2 * EPS * abs( e );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[ i ]+'. y: '+y+'. E: '+e+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the inverse cotangent for huge positive values', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	x = hugePositive.x;
+	expected = hugePositive.expected;
+	for ( i = 0; i < x.length; i++ ) {
+		y = acotf( x[ i ] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[ i ]+'. E: '+e+'.' );
+		} else {
+			delta = abs( y - e );
+			tol = 1.2 * EPS * abs( e );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[ i ]+'. y: '+y+'. E: '+e+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the inverse cotangent for huge negative values', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	x = hugeNegative.x;
+	expected = hugeNegative.expected;
+	for ( i = 0; i < x.length; i++ ) {
+		y = acotf( x[ i ] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[ i ]+'. E: '+e+'.' );
+		} else {
+			delta = abs( y - e );
+			tol = 1.2 * EPS * abs( e );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[ i ]+'. y: '+y+'. E: '+e+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `NaN`', function test( t ) {
+	var v = acotf( NaN );
+	t.equal( isnanf( v ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns `+0` if provided `+infinity`', function test( t ) {
+	var v = acotf( PINF );
+	t.equal( isPositiveZerof( v ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns `-0` if provided `-infinity`', function test( t ) {
+	var v = acotf( NINF );
+	t.equal( isNegativeZerof( v ), true, 'returns expected value' );
+	t.end();
+});
```

Comment:
Assuming this is copied over and a wrapper `float64ToFloat32` is used.

Reply:
Yes.